jerry-core/parser/js/js-lexer.c

```diff
@@ -915,7 +915,7 @@ lexer_parse_string (parser_context_t *context_p, /**< context */
                     lexer_string_options_t opts) /**< options */
 {
 #if ENABLED (JERRY_ES2015)
-  size_t raw_length_dec = 0;
+  int32_t raw_length_dec = 0;
```

Comment:
What about calling this raw_length_adjusment, raw_length_diff, or raw_length_change?

Reply:
Sure! I've updated the pr.